Scroller/EditableWrapper.js

```diff
@@ -687,20 +682,45 @@ const EditableWrapper = (props) => {
 	}, [completeEditingByKeyDown]);
 
 	const handleGlobalKeyDownCapture = useCallback((ev) => {
-		if (getPointerMode() && !scrollContainerRef.current.contains(Spotlight.getCurrent()) && (mutableRef.current.selectedItem || mutableRef.current.focusedItem)) {
+		const {focusedItem, selectedItem} = mutableRef.current;
+
+		// If the pointer mode is `true` and the focused component is not contained in scrollContainerRef,
+		// only `handleGlobalKeyDownCapture` is called instead of `handleKeyDownCapture`
```

Comment:
It'd be better to add
```suggestion
		// only `handleGlobalKeyDownCapture` is called instead of `handleKeyDownCapture`
		// Below is mainly for handling key pressed while pointer mode is `true`.
```

Reply:
It's fixed.